data/json/items/armor/storage.json

```diff
@@ -1612,6 +1621,15 @@
         "max_item_length": "30 cm",
         "magazine_well": "800 ml",
         "moves": 120
+      },
+	  {
```

Comment:
```suggestion
	    {
```

Reply:
you normally need to lint your files but it looks like these were just copy pasted and the indenting got messed up